programs/libraries/pretty_print.sc

```diff
@@ -0,0 +1,129 @@
+// Adds a pretty_print function which is equivalent to the print function, but prints stuff different colors depending on type. Avoids confusion between null and 'null' and between maps, strings, and nbts
+
+// Types & colors
+// null - purple
+// number - blue
+// string - orange
+// list - yellow
+// map - red
+// iterator - Darker yellow
+// function - magenta
+// task - grey
+// block - green
+// entity - Darker red
+// nbt - cyan
+// text - Unchanged, usually formatted already
+
+formatType(v) -> (
```

Comment:
generally scarpet names are snake case, not camel case (so `format_type` instead of `formatType`). I believe this is a convention across statically typed languages, though I may be mistaken. Certainly in scarpet, in-built functions must be snake-case

Reply:
Yeah you're right, I should change it.